src/main/data/studios.json

```
@@ -2171,6 +2171,16 @@
     "description": "Kominek Motion Pictures",
     "url": "https://kominek.ca"
   },
+  {
+    "code": "KO",
+    "contact": {
+      "address": "141 Cove Road, Oyster Bay, NY, 11771 USA",
```

```suggestion
      "address": "141 Cove Road, Oyster Bay, NY 11771 USA",
```

No comma between the state and the ZIP code.